src/EventStore.Core/VNodeBuilder.cs

```diff
@@ -1454,7 +1466,8 @@ public ClusterVNode Build(IOptions options = null, IPersistentSubscriptionConsum
                     _skipIndexScanOnReads,
                     _reduceFileCachePressure,
                     _initializationThreads,
-                    _faultOutOfOrderProjections);
+                    _faultOutOfOrderProjections,
+					_autoMergeIndexes);
```

Comment:
fix indentation

Reply:
I can't see the indentation issues in my editor